pull-request/src/person.js

```diff
@@ -7,9 +7,11 @@ function Person(first, last) {
 }
 
 Person.prototype.fullname = function() {
+    console.log("About to return full name");
     return this.firstName + " " + this.lastName;
 }
 
 Person.prototype.wishHappyBirthday = function() {
+    console.log("About to wish someone a happy birthday!");
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this logging statement. ([javascript:S2228](http://localhost:9067/coding_rules#rule_key=javascript%3AS2228))

[See it in SonarQube](http://localhost:9067/project/issues?id=training%3Apull-request&pullRequest=41&issues=AWgYK5PmUHOAr7vtanyr&open=AWgYK5PmUHOAr7vtanyr)

[ISSUE_KEY]: AWgYK5PmUHOAr7vtanyr